src/openklant/components/contactmomenten/api/filters.py

```diff
@@ -71,6 +88,10 @@ class Meta:
             "initiatiefnemer": ["exact"],
             "medewerker": ["exact"],
             "ordering": ["exact"],
+            "medewerker_identificatie__identificatie": ["exact"],
+            "medewerker_identificatie__achternaam": ["exact"],
+            "medewerker_identificatie__voorletters": ["exact"],
+            "medewerker_identificatie__voorvoegsel_achternaam": ["exact"],
```

Comment:
@alextreme je kunt hier niet op meer dan 1 waarde filteren, is dit een probleem denk je?

Reply:
Nee lijkt me niet, prima zo wat mij betreft